src/main/java/net/dv8tion/jda/internal/entities/PermissionOverrideImpl.java

```diff
@@ -103,12 +105,21 @@ public JDA getJDA()
         return api;
     }
 
+    @Nonnull
+    @Override
+    public IPermissionHolder getPermissionHolder()
+    {
+        return (IPermissionHolder) (role ? getRole() : getMember());  // permissionHolder is no longer a field
+    }
+
+    @Nullable
     @Override
     public Member getMember()
     {
         return getGuild().getMemberById(id);
     }
 
+    @Nullable
```

Comment:
Adding these to implementations is redundant.

Reply:
I guess I can then remove all Nullable / Nonnull annotations then? Because there were already a lot of Nonnulls but no Nullable ones so I added them.

Reply:
The nonnull is not redundant, nullable is the default so they are not needed.

Reply:
All methods are marked as Nonnull / Nullable in the interface already, aren't the Nonnulls redundant as well then?

Reply:
No, they indicate to the implementation writer that returning null is an error.